docs/src/HowToGuides/Atmos/Microphysics.md

```diff
@@ -1,125 +1,277 @@
+  - `q_sno` - snow specific humidity.
+
+## Assumed particle size relationships
+
+Particles are assumed to follow the mass(radius), cross section(radius), terminal velocity(radius)
+relationships defined as power laws.
+The coefficients are defined in `MicrophysicsParameters` module and are shown in the Table below.
+For rain and ice they correspond to spherical liquid water drops and ice particles, respectively.
+There is no assumed particle shape for snow and the relationships are based on empirical values
+from [Grabowski\_1998](https://journals.ametsoc.org/doi/full/10.1175/1520-0469%281998%29055%3C3283%3ATCRMOL%3E2.0.CO%3B2 ).
 
-|    symbol            |         definition                                        | units                    | default value         |
-|----------------------|-----------------------------------------------------------|--------------------------|-----------------------|
-|``n_{0_{MP}}``        | rain drop size distribution parameter                     | ``\frac{1}{m^4}``        | ``16 \cdot 10^6``     |
-|``\tau_{cond_evap}``  | cloud water condensation/evaporation timescale            | ``s``                    | ``10``                |
-|``\tau_{acnv}``       | cloud to rain water autoconversion timescale              | ``s``                    | ``10^3``              |
-|``q_{liq\_threshold}``| cloud to rain water autoconversion threshold              | -                        | ``5 \cdot 10^{-4}``   |
-|``E_{col}``           | collision efficiency between rain drops and cloud droplets| -                        | ``0.8``               |
-|``C_{drag}``          | rain drop drag coefficient                                | -                        | ``0.55``              |
-|``a_{vent}, b_{vent}``| rain drop ventilation factor coefficients                 | -                        | ``1.5 \;``,``\; 0.53``|
-|``K_{therm}``         | thermal conductivity of air                               | ``\frac{J}{m \; s \; K}``| ``2.4 \cdot 10^{-2}`` |
-|``\nu_{air}``         | kinematic viscosity of air                                | ``\frac{m^2}{s}``        | ``1.6 \cdot 10^{-5}`` |
-|``D_{vapor}``         | diffusivity of water vapor                                | ``\frac{m^2}{s}``        | ``2.26 \cdot 10^{-5}``|
-
-## Rain drop size distribution
-
-The rain-drop size distribution is assumed to follow Marshall-Palmer distribution (Marshall Palmer 1948 eq. 1):
 ```math
-\begin{equation}
-n(r) = n_{0_{MP}} exp\left(- \lambda_{MP} \, r \right)
-\end{equation}
+m(r) = \alpha r^{\beta}
+```
+```math
+a(r) = \gamma r^{\delta}
+```
+```math
+v_{term}(r) = \zeta r^{\eta}
 ```
 where:
- - ``r`` is the drop radius,
- - ``n_{0_{MP}}`` and ``\lambda_{MP}`` are the Marshall-Palmer distribution parameters (twice the values used in the Marshall Palmer 1948, because we use drop radius and not diameter).
+ - ``r`` is the particle radius,
+ - ``\alpha, \, \beta, \, \gamma, \, \delta, \, \zeta, \, \eta \,`` are the coefficients.
```

Comment:
Wouldn't it be clearer to call the exponents something like `exp_mass`, `exp_xsection`, `exp_velocity`? Then, nondimensionalize `r` with a characteristic scale `r0` in all expressions, and instead of `alpha` etc. introduce `mass_r0`, `xsection_r0` etc?

Reply:
done